src/splitInteger.test.js

```diff
@@ -4,18 +4,22 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(1, 1))
```

Comment:
```suggestion
  expect(splitInteger(1, 1))
```
you check here the same logic as in the these below

Reply:
fixed